package.json

```diff
@@ -31,8 +31,8 @@
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.6.0",
     "babel-preset-react": "^6.24.1",
-    "enzyme": "^3.2.0",
-    "enzyme-adapter-react-16": "^1.1.0",
+    "enzyme": "^3.4.0",
+    "enzyme-adapter-react-16.3": "^1.0.0",
```

Comment:
Are these changes required for this?  If so we need to add a request to update these packages downstream.

Reply:
works locally without these.